src/mesh/abstract_tree.jl

```diff
@@ -153,14 +163,28 @@ local_leaf_cells(t::AbstractTree) = leaf_cells(t)
 count_leaf_cells(t::AbstractTree) = length(leaf_cells(t))
 
 
+@inline function cell_coordinates(t::AbstractTree{NDIMS}, cell) where {NDIMS}
+  SVector(ntuple(d -> t.coordinates[d, cell], Val(NDIMS)))
+end
+
+@inline function set_cell_coordinates!(t::AbstractTree{NDIMS}, coords, cell) where {NDIMS}
```

Comment:
I am not sure if the `cell_` is required in the name, since its clear from the use what kind of coordinates are concerned. I assume that you did this to remain symmetrical with `child_coordinates`? Hm, I don't know, maybe just leave it as it is...

Reply:
Yeah, I had `child_coordinates` in mind and wanted to distinguish it. But I don't have a strong opinion here, so you can decide.

Reply:
Let's leave it as it is for now, since I don't have a strong opinion here either.